nexus/src/app/rack.rs

```diff
@@ -360,9 +360,9 @@ impl super::Nexus {
 
         // TODO
         // configure rack networking / boundary services here
-        // Currently calling some of the apis directly, but should we be using sagas
-        // going forward via self.run_saga()? Note that self.create_runnable_saga and
-        // self.execute_saga are currently not available within this scope.
+        // Currently calling some of the apis directly, but should we be using
+        // sagas going forward via self.sagas.saga_execute()? Note that
+        // this may not be available within this scope.
```

Comment:
I checked with @rcgoodfellow (who wrote this comment) and we weren't sure what this comment meant about these facilities not being available here.  The higher level point is presumably still valid (that we should consider using a saga here) so I left the comment mostly alone.